### ISSUE TITLE
The bot keeps sending duplicates of its PRs after they have been merged

### ISSUE THREAD

**Issue by jtojnar** [ryantm/nixpkgs-update#269]
Seems like the check whether the version exists in some branch is broken and the bot opens a new PR after we merge its original one. 

Most recent example: https://github.com/NixOS/nixpkgs/pull/129524

**Comment by jtojnar** [comment]
I guess this is the culprit:

https://github.com/ryantm/nixpkgs-update/blob/0c36c9b59719e14c3ee02fa7019063425e422e70/src/Update.hs#L223-L226

The packages with duplicates do have update script.

**Comment by ryantm** [comment]
Yes, this is a problem, but we need to check it later on after we know which version the updateScript tells us it wants to update too.
